libraries/AP_DDS/README.md

```diff
@@ -230,7 +230,45 @@ Next, follow the associated section for your chosen transport, and finally you c
   ```
   In order to consume the transforms, it's highly recommended to [create and run a transform broadcaster in ROS 2](https://docs.ros.org/en/humble/Concepts/About-Tf2.html#tutorials).
 
-
+## Using ROS 2 services (with Integration Services)
+
+### Prerequisites
+- Install and setup [Micro-XRCE Agent](https://micro-xrce-dds.docs.eprosima.com/en/latest/installation.html#installing-the-agent-standalone)
+- Install and setup [Integration Services](https://micro-xrce-dds.docs.eprosima.com/en/latest/installation.html#installing-the-client-standalone) (it would be good to have a seperate workspace for this)
+  - Get System Handles for [ROS 2](https://github.com/eProsima/ROS2-SH)
+  - Get System Handles for [Fast-DDS](https://github.com/eProsima/FastDDS-SH)
+
+### Setup
+- Clone the [ardupilot_ros2](https://github.com/arshPratap/ardupilot_ros2) repo in your ROS 2 workspace
+- Switch to **ArmingClient** branch
+- Build the [ap_custom_services](https://github.com/arshPratap/ardupilot_ros2/tree/ArmingClient/ap_custom_services) package
+- Build the [ap_service_clients](https://github.com/arshPratap/ardupilot_ros2/tree/ArmingClient/ap_service_clients) package
+- In a new terminal, successfully source ROS 2 installation
+- Move to your ROS 2 workspace and source the installation (you should now see the above 2 pacakges when you run `ros2 pkg list`)
+- Move to your Integration Service workspace and use the following command to successfully link the ***ap_custom_services*** package with Integration Services `colcon build --cmake-args -DMIX_ROS_PACKAGES="ap_custom_services"`
```

Comment:
Update this section to reflect that the message and service interfaces are defined in `ardupilot_msgs` (https://github.com/ArduPilot/ardupilot/pull/24277).

Reply:
Done